modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/application/resource/test/HeadlessBuilderResourceTest.java

```diff
@@ -567,15 +567,15 @@ private void _addAggregationObjectField(
 	public void testPutByExternalReferenceCode() throws Exception {
 		_addAPIApplication(
 			_API_APPLICATION_ERC_1, _API_ENDPOINT_ERC_1, _BASE_URL_1,
-			_objectDefinition1.getExternalReferenceCode(),
+			_objectDefinitionJSONObject1.getString("externalReferenceCode"),
 			_API_APPLICATION_PATH_1);
 
 		_publishAPIApplication(_API_APPLICATION_ERC_1);
 
 		_assertSuccessfulHttpCode(
 			HTTPTestUtil.invokeToHttpCode(
 				JSONUtil.put(
-					"applicationStatus", "unpublished"
+					"applicationStatus", "published"
 				).put(
 					"baseURL", "test"
```

Comment:
Can't we use _BASE_URL_2 here as well instead of test? 🤔

Reply:
Sure, I'll update to the exsiting _URL_2

Reply:
It's just to align the design of both tests :D

Reply:
that's great, thanks!